src/expected_loglik.jl

```diff
@@ -75,7 +75,7 @@ function expected_loglik(
     # size(fs): (length(y), n_points)
     fs = √2 * std.(q_f) .* xs' .+ mean.(q_f)
     lls = loglikelihood.(lik.(fs), y)
-    return sum((1 / √π) * lls * ws)
+    return (1 / √π) * mapreduce(dot, +, eachrow(lls), ws)
```

Comment:
```suggestion
    return mapreduce(dot, +, eachrow(lls), ws) / √π
```
But also, at this point it's getting really hard to glance at the code and understand what's going on.... what do you think of adding a comment to explaining that?